pyat/at/collective/beam_loading.py

```diff
@@ -256,7 +282,11 @@ def __init__(self, family_name: str, length: float, voltage: float,
         self._vbeam_phasor = numpy.zeros(2)
         self._vbeam = numpy.zeros(2)
         self._vgen = numpy.zeros(2)
-        self._vcav = numpy.array([self.Voltage,
+        
+        
+        cavity_voltage = self._passive_vset if self._cavitymode==3 else self.Voltage 
```

Comment:
I don't think this will pass the pep8 test.....
You may write (it is also better to you your enum definition rather than int:
cavity_voltage = self.Voltage
if(self._cavitymode==CavityMode.PASSIVEVOLTAGE):
    cavity_voltage = self._passive_vset